### DISCUSSION TITLE
Tabs & Columns?

### DISCUSSION THREAD

**Discussion by CryptoMares** [spacedriveapp/spacedrive#1660] [Feature Request]
Would be great to have those multitasking features.

**Comment by utkubakir** [comment]
Tabs are planned, what do you mean by columns?

**Reply by CryptoMares** [reply]
It's a parallel window, like in OneCommander, Total Commander, Norton Commander, Far, etc. It's surprisingly handy to have 2 windows in one, side by side. Just try out OneCommander and you will see.
